client/components/SampleCensus.scss

```diff
@@ -153,6 +161,19 @@
           }
         }
 
+        &__answer-format {
+          &__label {
+            font-size: 20px;
+            margin: 30px 0 16px 0;
+          }
+
+          &__box {
+            background: #F2F2F2;
+            border: 1px solid #E5E5E5;
+            padding: 21px 28px;
```

Comment:
I think we either want to have different styles for the desktop / mobile padding or maybe use a viewport measurement instead?